main/zoe/guide/contracts/covered-call.md

```diff
@@ -2,160 +2,227 @@
+const invitationIssuer = E(zoe).getInvitationIssuer();
+const bobExclOption = await invitationIssuer.claim(coveredCall);
+```
+
+Let's imagine that Bob wants to sell the invitation.  He can start a swap
+instance to trade this invitation for bucks.
 
-// Let's imagine that Bob wants to sell the invite for the call option.
-// He can create a swap to trade this invite for bucks.
+```js
 const swapIssuerKeywordRecord = harden({
-  Asset: inviteIssuer,
+  Asset: invitationIssuer,
   Price: bucksR.issuer,
 });
-const bobSwapInvite = await zoe.makeInstance(swapInstallationId, swapIssuerKeywordRecord);
+const bobSwapSeat =
+  await E(zoe).startInstance(swapInstallation, swapIssuerKeywordRecord);
+```
+  
+Bob specifies that he wants to swap the invitation for 1 buck, and escrows
+the covered call invitation. In exchange, he gets a swap invitation he can
+share.
 
-// Bob wants to swap the invite with the same units as his
-// current invite from Alice. He wants 1 buck in return.
+```js
 const bobProposalSwap = harden({
-  give: { Asset: inviteIssuer.getAmountOf(bobExclOption) },
+  give: { Asset: invitationIssuer.getAmountOf(bobExclOption) },
   want: { Price: bucks(1) },
 });
 
 const bobPayments = harden({ Asset: bobExclOption });
+const bobSwapSeat = await E(zoe).offer(bobSwapInvitation, bobProposalSwap, bobPayments);
 
-// Bob escrows his option in the swap and gets an invite as the outcome that he can send to Dave
-const { outcome: bobSwapInviteP, payout: bobPayoutP } = await E(zoe).offer(
-  bobSwapInvite,
-  bobProposalSwap,
-  bobPayments,
-);
-
-const daveSwapInvite = await bobSwapInviteP;
+const daveSwapInvitation = bobSwapSeat.getOfferResult();
 ```
 
-Bob now has a call option, in the form of an invite, ready to sell via an atomic swap contract.
+Bob now has a call option that he can sell via an atomic swap contract.
 
 ## Buying An Option
 
-Another user, let's call him Dave, is looking to buy the option to trade his 7 simoleans for 3 moola, and is willing to pay 1 buck for the option. He is interested in Bob's swap invite so he checks that this instance matches what he wants. He can check the
-balance of the invite to see what contract it is for, and any
-contract-provided information about what the invite can be used for.
+Another user, let's call him Dave, is looking to buy the option to trade
+his 7 simoleans for 3 moola, and is willing to pay 1 buck for the
+option. He is interested in Bob's swap invitation so he checks that this
+instance matches what he wants. He can check the amount of the invitation
+to see what contract it is for, and any contract-provided information about
+what the invitation can be used for.
 
 ```js
-// Check the balance
 const {
-  value: [{ instanceHandle: swapInstanceHandle }],
-} = inviteIssuer.getAmountOf(daveSwapInvite);
-
-const {
-  installationHandle: daveSwapInstallId,
-  issuerKeywordRecord: daveSwapIssuers,
-} = zoe.getInstance(swapInstanceHandle);
+  installation: daveSwapInstall,
+  instance,
```

Comment:
do we need the instance?

Reply:
Dave should validate the install and the issuers. I've added that.